message_test.go

```diff
@@ -115,7 +115,7 @@ func TestCanParseWHISPERActionMessage(t *testing.T) {
 }
 
 func TestCanParsePRIVMSGMessage(t *testing.T) {
-	testMessage := "@badges=premium/1;color=#DAA520;display-name=FletcherCodes;emotes=;flags=;id=6efffc70-27a1-4637-9111-44e5104bb7da;mod=0;room-id=408892348;subscriber=0;tmi-sent-ts=1551473087761;turbo=0;user-id=269899575;user-type= :fletchercodes!fletchercodes@fletchercodes.tmi.twitch.tv PRIVMSG #clippyassistant :Chew your food slower... it's healthier"
+	testMessage := "@badges=premium/1;color=#DAA520;display-name=FletcherCodes;emotes=;first-msg=1;flags=;id=6efffc70-27a1-4637-9111-44e5104bb7da;mod=0;room-id=408892348;subscriber=0;tmi-sent-ts=1551473087761;turbo=0;user-id=269899575;user-type= :fletchercodes!fletchercodes@fletchercodes.tmi.twitch.tv PRIVMSG #clippyassistant :Chew your food slower... it's healthier"
```

Comment:
I'd love a second test for this to ensure the negative case as well

Reply:
I thought about this, but was unsure if you'd be okay with expanded/cluttered cases.

Reply:
Yes I would! Best is a Separate Test that has a speakable name. Like `TestCanParseFirstMessage` or something like that. I'd rather have more test functions than a few functions that assert too much at once

Reply:
So what I did here (didn't see the comment, sorry!) was moved the entire test to be TDD similar as you do in another function. I figure it's probably easier to add functionality down the line during TDD than managing a ton of different functions.